### ISSUE TITLE
Warning Ratio does not equal 1.0

### ISSUE THREAD

**Issue by Bullitt33** [scfmod/FS22_MaterialProcessor#1]
Hello,

Getting a warning in the log stating that the mixing ratio does not equal 1.0. I am pretty good a math but maybe I am missing something here.

Here is how I have it set up and it will not mix.   Not sure how 0.9 +0.1 does not equal 1.0

materialProcessor type="blend" needsToBeTurnedOn="true"
        configurations
            configuration name="Herbicide Mix" litersPerSecond="500"
                output fillType="HERBICIDE" fillUnitIndex="3"
                    input fillType="WATER" fillUnitIndex="1" ratio="0.9" 
                    input fillType="HERBICIDEPURE" fillUnitIndex="2" ratio="0.1" 
                output
            configuration

        configurations
    materialProcessor

**Comment by scfmod** [comment]
Hm maybe I forgot to update the ratio check code, I'll look into it :-) Just ignore the warning for now and i'll update it this weekend, and thanks for reporting!

**Comment by Bullitt33** [comment]
So doings some more work on this last night that there seems to be some sort of error in your scripting. I finally got this working by messing the the ratio numbers. The first step I moved the ratios back to 0.5 each. It worked! I still got the warning for Ratio does not equal 1.0.  I then changed the ratios back to 0.9 and 0.1 and it stopped working again and the same error remained.

After going back and forth the furthest down I could take the ratio's is 0.85 and 0.15. The Warning still remains for not equaling 1.0 but it does work as intended! May be something to look at. Other than that it seems to work great so far and is a great lua script that definitely needs to make an early appearance in FS25!
